cadc-vos-server/src/main/java/ca/nrc/cadc/vos/server/NodeDAO.java

```diff
@@ -2718,7 +2723,7 @@ void setValues(PreparedStatement ps)
             sb.append(pval);
             sb.append(",");
 
-            log.debug("setValues:" + sb.toString());
+            //log.debug("setValues:" + sb.toString());
```

Comment:
did you mean to leave this commented out?

Reply:
Yes there is the same debug a bit further down which logs the entire preparedstatement.